packages/flutter/pubspec.yaml

```diff
@@ -30,14 +30,17 @@ dev_dependencies:
   crypto: 2.0.2+1 # TRANSITIVE DEPENDENCY
   csslib: 0.14.1 # TRANSITIVE DEPENDENCY
   dart_style: 1.0.8 # TRANSITIVE DEPENDENCY
+  file: 2.3.4 # TRANSITIVE DEPENDENCY
```

Comment:
This doesn't seem right. The `flutter` package doesn't depend on `file` in any way, directly or indirectly.

Reply:
This is a `dev_dependency`. `flutter` depends on `flutter_test`, which depends on `file`.